src/lib/signature_lib/test/signature_lib_tests.ml

```diff
@@ -0,0 +1,39 @@
+open Signature_lib
+
+let%test_module "Signatures are unchanged test" =
```

Comment:
These is a great test to add!

Would it be possible to quickly make it actually use our standard set of signature test vectors?


https://github.com/o1-labs/proof-systems/blob/bbc4c77382507c3a0c66f9fda1a7499a5d547b7a/signer/tests/signer.rs#L125